pages/installation/operating_system_packages.rst

```diff
@@ -64,7 +64,9 @@ Debian 8, Ubuntu 16.06 systemd     ``sudo systemctl enable graylog-server``
 Update to latest Version
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-If you like to install the latest Graylog Version as Update and you had used the repository package before just install the latest repository package and use your distribution command to update the ``graylog-server`` package::
+When you had used the repository package before to install Graylog, you need to update the repository package at first. The new repository package will update and replace the repository URL. Without this update you will only get Bugfix releases of your installed Version.
```

Comment:
"you need to update the repository package at first." &rarr; "you first need to update the repository package."